biowasm.json

```
@@ -399,6 +399,17 @@
 					"branch": "a12da33"
 				}
 			]
+		},
+		{
+			"name": "muscle",
+			"programs": ["muscle"],
+			"description": "MUSCLE is widely-used software for making multiple alignments of biological sequences",
```

Let's replace this with:
```suggestion
			"description": "Multiple alignments of biological sequences",
```